flang/tools/flang-tidy/FlangTidyCheck.h

```diff
@@ -493,6 +495,9 @@ class FlangTidyCheck : public semantics::BaseChecker {
     str.append(" [%s]");
     parser::MessageFixedText newMessage{str.c_str(), str.length(),
                                         message.severity()};
+    if (warningsAsErrors_) {
```

Comment:
For some reason `warningsAsErrors_` is always `false` here. I don't see why, though.